src/main/java/utilities/CalendarManager.java

```diff
@@ -0,0 +1,158 @@
+     * @return object type Date
+     */
+    private boolean isRightDateAccordingConventions(final String input) {
+        try {
+            String[] divide = input.split(" ", 2);
+            dates = Integer.parseInt(divide[0]);
+            String fillUpLowBar = divide[1].replace(' ', '_');
+            dateFormat = DateFormatInput.valueOf(fillUpLowBar);
+            if ((dateFormat.isSingular() && dates != 1) | (!dateFormat.isSingular() && dates == 1) | dates < 0) {
+                return false;
+            }
+            if (fillUpLowBar.contains("ago")) {
+                dates = -dates;
+            }
+            return true;
+        } catch (NumberFormatException e) {
+            return false;
+        }
+    }
+
+    /**
+     * Convert a string to data.
+     * Only for 3 word, TODAY, YESTERDAY and TOMORROW.
+     *
+     * @param input input is value which will convert to date
+     * @return object type Date
+     */
+    private boolean isOnlyWord(final String input) {
+        generateDateActual();
+        if (input.equals("TODAY")) {
+            dates = 0;
+            dateFormat = DateFormatInput.valueOf(input);
+            return true;
+        }
+
+        if (input.equals("YESTERDAY")) {
+            dates = -1;
+            dateFormat = DateFormatInput.valueOf(input);
+            return true;
+        }
+
+        if (input.equals("TOMORROW")) {
+            dates = 1;
+            dateFormat = DateFormatInput.valueOf(input);
+            return true;
+        }
+        return false;
+    }
+
+    /**
+     * Generates actual date.
+     */
+    private void generateDateActual() {
+        java.text.DateFormat dateFormat = new SimpleDateFormat("MM/dd/yyyy HH:mm:ss");
+        date = new Date();
+        dateActualComplete = dateFormat.format(date);
+        setCalendarFormat();
+    }
+
+    /**
+     * Sets calendar according of string.
+     */
+    private void setCalendarFormat() {
+        calendarDate = Calendar.getInstance();
+        calendarDate.setTime(date);
+    }
+
+    /**
+     * Sets date according requirement of string.
+     */
+    private void managerInputs() {
+        if ("year" == dateFormat.getValue()) {
+            calendarDate.add(Calendar.YEAR, dates);
+        }
+        if ("month" == dateFormat.getValue()) {
+            calendarDate.add(Calendar.MONTH, dates);
+        }
+        if ("date" == dateFormat.getValue()) {
+            calendarDate.add(Calendar.DATE, dates);
+        }
+        if ("hour" == dateFormat.getValue()) {
+            calendarDate.add(Calendar.HOUR, dates);
+        }
+        if ("minute" == dateFormat.getValue()) {
+            calendarDate.add(Calendar.MINUTE, dates);
+        }
+        if ("second" == dateFormat.getValue()) {
+            calendarDate.add(Calendar.SECOND, dates);
+        }
+    }
+
+    /**
+     * Gets result of string input.
+     *
+     * @return a date object.
+     */
+    private Date result() {
```

Comment:
Maybe it could be a better name "getResult"

Reply:
I will change